Mirror/Runtime/NetworkReader.cs

```diff
@@ -265,6 +265,40 @@ public NetworkIdentity ReadNetworkIdentity()
             return go.GetComponent<NetworkIdentity>();
         }
 
+        public NetworkBehaviour ReadNetworkBehaviour()
+        {
+            uint netId = ReadPackedUInt32();
+            if (netId == 0)
+            {
+                return null;
+            }
+            GameObject go;
+            if (NetworkServer.active)
+            {
+                go = NetworkServer.FindLocalObject(netId);
+            }
+            else
+            {
+                go = ClientScene.FindLocalObject(netId);
```

Comment:
please use NetworkIdentity.spawned.TryGetValue instead of FindLocalObject. also avoids GetComponent call below.